blogpost.css

```diff
@@ -1,272 +0,0 @@
-* {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
-
-body {
-    display: grid;
-    max-width: 100vw;
-    height: auto;
-    margin: 10px 50px 20px 50px;
-    grid-template-rows: 10rem 50rem 1fr 30rem;
-    gap: 0.3rem 1rem;
-    grid-template-areas:
-    'header'
-    'main'
-    'footer-container';
-    background-color: #ffffff;
-    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100' height='100' viewBox='0 0 100 100'%3E%3Cg stroke='%23CCC' stroke-width='0' %3E%3Crect fill='%23F5F5F5' x='-60' y='-60' width='110' height='240'/%3E%3C/g%3E%3C/svg%3E");
-}
-
-  /**Header**/
-
-  #header {
-    display: grid;
-    max-width: 100vw;
-    grid-area:header;
-    grid-template-rows: 10rem;
-    grid-template-areas: 
-    'header';
```

Comment:
If you're doing a one dimensional layout flexbox might be an easier tool to use here. You can put flex boxes inside grids just fine (and vice versa)